tests/testthat/test-fanout.R

```diff
@@ -84,6 +84,7 @@ test_that("sims expansion is correct",{
 
 
 test_that("fan_out ids are correct",{
+  sleep_large <- resample_data(sleep, N = 200)
```

Comment:
is this used anywhere?

Reply:
that test fails intermittently, was trying to see if upping sample size would help

Reply:
oic I didn't actually implement. commit to follow